controllers/grid/queries/QueryNotesGridHandler.php

```diff
@@ -246,7 +245,7 @@ public function getCanManage($note)
         if ($note === null) {
             return $isAdmin;
         } else {
-            return ($note->getUserId() == $this->_user->getId() || $isAdmin);
+            return ($note->userId == $this->_user->getId() || $isAdmin);
         }
```

Comment:
Hmm, the UI of GitHub is bugged here, but the if/else can be shortened into:

```suggestion
return $note?->userId == $this->_user->getId() || $isAdmin;
```